backend/typescript/services/implementations/__tests__/volunteerService.test.ts

```diff
@@ -122,6 +152,33 @@ describe("Testing VolunteerService Functions", () => {
     });
   });
 
+  it("updateVolunteerById", async () => {
+    const mockUpdateVolunteerDTO = {
+      userId: "1",
+      status: Status.APPROVED,
+    };
+
+    await volunteerService.updateVolunteerById("1", mockUpdateVolunteerDTO);
+
+    const res = await volunteerService.getVolunteerByID("1");
+
+    expect(res).toMatchObject(testUpdatedUserVolunteers[0]);
+  });
+
+  it("updateVolunteerByUserId", async () => {
+    const mockUpdateVolunteerDTO = {
+      id: "2",
+      userId: "2",
+      status: Status.REJECTED,
+    };
+
+    await volunteerService.updateVolunteerByUserId("2", mockUpdateVolunteerDTO);
+
+    const res = await volunteerService.getVolunteerByID("2");
+
+    expect(res).toMatchObject(testUpdatedUserVolunteers[1]);
```

Comment:
i also changed this to [1] because i think that was what fit the code? lmk if im wrong